src/Xamarin.Android.Build.Tasks/Tasks/GetAdditionalResourcesFromAssemblies.cs

```diff
@@ -230,7 +231,7 @@ string MakeSureLibraryIsInPlace (string destinationBase, string url, string vers
 						client.DownloadFileTaskAsync (url, file).Wait (Token);
 						LogMessage ("  Downloading Complete");
 					} catch (Exception e) {
-						LogCodedError ("XA5208", "Download failed. Please download {0} and put it to the {1} directory.", url, destinationDir);
+						LogCodedError ("XA5208", "Download failed. Please download {0} and copy it to {1}", url, file);
```

Comment:
I think as general policy we should "quote" filename paths. I'm partial to backticks at present, but any form of quoting would be good:

    XA5208: Unable to download `url`. Please download and copy into `directory`.